modules/aerodyn/src/FVW.f90

```diff
@@ -1572,8 +1572,9 @@ subroutine UA_Init_Wrapper(AFInfo, InitInp, interval, p, x, xd, OtherState, m, E
    type(FVW_MiscVarType),           intent(inout)  :: m           !< Initial misc/optimization variables
    integer(IntKi),                  intent(  out)  :: ErrStat     !< Error status of the operation
    character(*),                    intent(  out)  :: ErrMsg      !< Error message if ErrStat /= ErrID_None
+   
+   integer(IntKi), parameter :: NumBladesPerWing = 1 !bjj why are we using this number?
```

Comment:
Good question.  I don't know why we are allocating these arrays as 2D.

Maybe there was a reason early in the development that is now irrelevant?

Reply:
This is introduced because UA is implemented for many blades at once, whereas OLAF has a list of wings, and we call UA for each of the wing.  The parameter is introduced instead of putting "1" in the source code. It's supposed to confuse less the reader, but it seems it was not too clear :) 

Maybe we can add in the comment: 
```fortran
NumBladesPerWing !< UA is for multiple blades, FVW calls one instance of UA per wing. 
```